README.md

```diff
@@ -205,4 +205,12 @@ Currently this is set up for benchmark runs which have ICs which are already gen
 
 Prototype 5 was run with global-workflow hash 005468b9299ea6fc9afdbeace33c336c6797833a
 
+## Managing Default / Adding new configuration variables
 
+workflow/default is the central place of managing default values of configuration variables. In principle, every single variable that could be configured by CROW need to have a default value included in workflow/default. A series of YAML files are used to storage these values. During the configuration step, CROW will load values from workflow/default first, and then customized by the overriding values in the YAML file selected by the user within workflow/cases/.
```

Comment:
"CROW need" -> "CROW needs" 
"to storage" -> "to store" 
"then customized" -> "then will be customized"

Reply:
OK. I will fix these.